package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/generator-app-excshell",
-  "version": "0.2.1",
+  "version": "0.2.2",
```

Comment:
everything else looks good, can we revert this version change though? it is automated by our publish action

Reply:
@shazron , ^^^ done